src/interface/raftex.thrift

```diff
@@ -46,6 +46,8 @@ enum ErrorCode {
     E_PERSIST_SNAPSHOT_FAILED = -13;
     E_RPC_EXCEPTION = -14;              // An thrift internal exception was thrown
     E_NO_WAL_FOUND = -15;
+    E_APPLY_FAIL = -16;
```

Comment:
Not used, maybe you miss it or will use later?

Reply:
Ah, it is used in `RaftPart.cpp`